lib/src/editorhelper.cpp

```diff
@@ -234,4 +235,255 @@ std::unique_ptr<uhdr_raw_image_ext_t> apply_resize(uhdr_raw_image_t* src, int ds
   }
   return std::move(dst);
 }
+
+const float BT601RGBtoYUVMatrix[5] = {0.299, 0.587, 0.114, 1.772, 1.402};
+
+const float BT709RGBtoYUVMatrix[5] = {0.2126, 0.7152, 0.0722, 1.8556, 1.5748};
+
+const float BT2020RGBtoYUVMatrix[5] = {0.2627, 0.6780, 0.0593, 1.8814, 1.4746};
+
+std::unique_ptr<uhdr_raw_image_ext_t> convert_raw_input_to_ycbcr(uhdr_raw_image_t* src) {
```

Comment:
Generally they look good to me. A few suggestions:

(1) I don't think the editorhelper is the best place for color conversion, ideally we should have a separate .h/.cpp file to keep them. (Or for now I think keep them in gainmapmath.cpp is a better choice)

(2) I think there are a lot of common code here and in gainmapmapth.h / .cpp, especially the parameters eg `BT601RGBtoYUVMatrix`. We'd better keep those parameters just in one place.

Reply:
Done